### PR TITLE
www.instiz.net

### PR DESCRIPTION
**Pull request by green1052**
1.  remove watermark
2.  remove blank
image

image

### CODE REVIEW

**Review by piquark6046 — CHANGES_REQUESTED** [review]
Please give me an example URL via email or PR comment.

**Comment by green1052** [comment]
https://www.instiz.net/name/48905543

**Comment by green1052** [comment]
sry 
https://www.instiz.net/pt/7132594?green=1

File: filters-share/extended_css_INJECTION.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
+www.instiz.net#$?#.backlogo { background-image: none !important; }
 5donews.co.kr#$?#.body > .content.xe_content:not(:has(article)) { width: auto !important; }
-mule.co.kr#$?#.section-lefttop-center:has(> .mule-today) { width: 100% !important; }
\ No newline at end of file
+mule.co.kr#$?#.section-lefttop-center:has(> .mule-today) { width: 100% !important; }
```
Comment (piquark6046) [inline filters-share/extended_css_INJECTION.txt]: This rule must be located in `AnnoyancesFilter/sections/tweaks.txt` of AdguardFilters.
https://github.com/AdguardTeam/AdguardFilters/pull/87489

File: filters-share/specific_ELEMHIDE.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
+www.instiz.net##.footad
```
Comment (piquark6046) [inline filters-share/specific_ELEMHIDE.txt]: Cloud not find.
However, It can be covert to `##div[class$="ad"]`.

**Review by piquark6046 — CHANGES_REQUESTED** [review]

File: filters-share/specific_ELEMHIDE.txt (hunk 2 above)
Comment (green1052) [reply filters-share/specific_ELEMHIDE.txt]: fix 3ad6cbb

File: filters-share/specific_ELEMHIDE.txt [hunk 3]
```diff
@@ -1,3 +1,4 @@
+www.instiz.net##.hot_under
```
Comment (piquark6046) [inline filters-share/specific_ELEMHIDE.txt]: Where?

<details>
<summary>Screenshot</summary>

Scree222nshot
Screenshot99

</details>

File: filters-share/extended_css_INJECTION.txt (hunk 1 above)
Comment (green1052) [reply filters-share/extended_css_INJECTION.txt]: deleted in 9a7fdd1

**Review by piquark6046 — CHANGES_REQUESTED** [review]

File: filters-share/specific_ELEMHIDE.txt (hunk 3 above)
Comment (green1052) [reply filters-share/specific_ELEMHIDE.txt]: 1. 
image

2.
image

Comment (piquark6046) [reply filters-share/specific_ELEMHIDE.txt]: I cloud not find.
Please share a screenshot contains the elements in `.hot_under`.

Comment (green1052) [reply filters-share/specific_ELEMHIDE.txt]: image
image

**Review by piquark6046 — APPROVED** [review]

**Pull request merged**